### PR TITLE
Add overlays to configuration schema

### PR DESCRIPTION
**Pull request by gjasny**
See https://github.com/microsoft/vcpkg/blob/master/docs/users/registries.md#configuration-overlay-ports

### CODE REVIEW

File: docs/vcpkg-configuration.schema.json [hunk 1]
```diff
@@ -20,6 +20,20 @@
       "type": "string",
       "description": "A message to print if this entry is evaluated."
     },
+    "overlay-ports": {
+      "type": "array",
+      "description": "An array of port overlay paths.",
+      "items": {
+        "type": "string"
```
Comment (BillyONeal) [inline docs/vcpkg-configuration.schema.json:27]:
```suggestion
        "$ref": "vcpkg-schema-definitions.schema.json#/definitions/port-name"
```

**Review by BillyONeal — COMMENTED** [review]
LGTM iff the suggestion is applied.

**Review by BillyONeal — APPROVED** [review]
Actually LGTM period, I forgot this is a full path not just a name

**Comment by BillyONeal** [comment]
Sorry I missed in https://github.com/microsoft/vcpkg-tool/pull/743

**Review by valeriaconde — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution to vcpkg!

**Pull request merged**